backend/app/api/endpoints/users.py

```
@@ -177,3 +177,72 @@ def get_team_info(team_id: int, is_system: bool) -> Optional[QuickAccessTeam]:
         teams=result_teams,
     )
+
+
+# ==================== Welcome Config (Slogan & Tips) ====================
+
+CHAT_SLOGAN_TIPS_CONFIG_KEY = "chat_slogan_tips"
+
+# Default slogan and tips configuration
+DEFAULT_SLOGAN_TIPS_CONFIG = {
+    "slogan": {
+        "zh": "欢迎使用 Wegent，让 AI 成为您的智能助手",
```

Ruff (0.14.8)

String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)